airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -52,6 +52,17 @@ class Config:
         default=[],
     )
 
+    action_report_time: str = Field(
+        title="action_report_time",
```

Comment:
Can you change this so that the title value matches the title values declared everywhere else? (Action Report Time instead of action_report_time)

Reply:
thanks for the catchup, done :heavy_check_mark: